source/iostreams.tex

```diff
@@ -9620,7 +9624,7 @@
 \pnum
 \remarks
 An
-implementation might well provide an overriding definition for this function
+implementation may provide an overriding definition for this function
 signature if it can determine that more characters can be read from the input
```

Comment:
```suggestion
signature if it can determine whether more characters can be read from the input
```
Whether we give permission for an override to be provided can't depend on whether there are characters in the buffer.

Reply:
Good point.